saleor/plugins/webhook/tasks.py

```diff
@@ -25,16 +31,19 @@ class WebhookSchemes(str, Enum):
     GOOGLE_CLOUD_PUBSUB = "gcpubsub"
 
 
-@app.task
-def trigger_webhooks_for_event(event_type, data):
+def _get_webhooks_for_event(event_type, webhooks=None):
+    """Get active webhooks from the database for an event."""
     permissions = {}
-    required_permission = WebhookEventType.PERMISSIONS[event_type].value
+    required_permission = WebhookEventType.PERMISSIONS.get(event_type)
     if required_permission:
-        app_label, codename = required_permission.split(".")
+        app_label, codename = required_permission.value.split(".")
         permissions["app__permissions__content_type__app_label"] = app_label
         permissions["app__permissions__codename"] = codename
 
-    webhooks = Webhook.objects.filter(
+    if webhooks is None:
+        webhooks = Webhook.objects.all()
```

Comment:
Should we filter out not active webhooks?

Reply:
It's done in line 47.